### DISCUSSION TITLE
Cannot read properties of null (reading 'scope')

### DISCUSSION THREAD

**Discussion by its-saniya-khan** [react-hook-form/react-hook-form#9696] [Q&A]
Hello,

I'm using react-hook-forms(v7.41.1) in my react-native(0.66.4) and everything was working fine. Suddenly, since yesterday I'm observing an error whenever I boot up my app(both ios and android)
`node_modules/react-hook-form/dist/index.cjs.js: /Users/UserName/AppName/node_modules/react-hook-form/dist/index.cjs.js: Cannot read properties of null (reading 'scope')

RCTFatal
__28-[RCTCxxBridge handleError:]_block_invoke
_dispatch_call_block_and_release
_dispatch_client_callout
_dispatch_main_queue_drain
_dispatch_main_queue_callback_4CF
__CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__
__CFRunLoopRun
CFRunLoopRunSpecific
GSEventRunModal
-[UIApplication _run]
UIApplicationMain
main
start_sim`

My node version is v18.12.1.
Thing I've tried
1. Downgrading and Upgrading my node
2. Cleaning the build folder and any residual cache
3. Setting initial value as {} for defaultValues

**Comment by leapful** [comment]
@its-saniya-khan 

Based on your stack track error message `Cannot read properties of null (reading 'scope')`, the error might be not from react-hook-form as there is no usage of any property or variable related to `scope`.

Please try to lookup in your code base files to see if any implementation that is using `scope` that can be null to solve the problem.

**Comment by realpxd** [comment]
have you found any solution for that? @its-saniya-khan
